# libhubbub: utils
# 1 files, 2 changed lines
#
# Regenerate: tools/regen-patches.sh && tools/split-patches.py
#
diff -urN upstream/libhubbub/src/utils/string.c browser/libhubbub/src/utils/string.c
--- upstream/libhubbub/src/utils/string.c
+++ browser/libhubbub/src/utils/string.c
@@ -9,7 +9,7 @@
 #include <inttypes.h>
 #include <stdbool.h>
 #include <string.h>
-#include "utils/string.h"
+#include "utils/string.h"
 
 
 /**
